cmd/config-current.go

```diff
@@ -138,6 +141,20 @@ func (s *serverConfig) GetStorageClass() (storageClass, storageClass) {
 	return s.StorageClass.Standard, s.StorageClass.RRS
 }
 
+// Calculates MD5 hash of the config file
```

Comment:
we should compute the config hash only on startup and after an `mc admin config set ...` operation. For every RPC call, we should return the hash computed previously.